commands/upgrade/__init__.py

```diff
@@ -113,6 +113,11 @@ def upgrade(args, breadcrumbs):
 
     if workflow.failure:
         sys.exit(1)
+    elif not args.resume:
+        sys.stdout.write(
+            'To continue with the upgrade reboot the system.'
```

Comment:
Nit: I'd rephrase it `Please reboot the system to continue with the upgrade` to emphasize the reboot action necessity.

Reply:
I like the suggested wording, I would just drop the please, so

```suggestion
            'Reboot the system to continue with the upgrade.'
```

but I am also ok with the current one. I will ask Chris about his opinion.

Reply:
@matejmatuska FYI,  I've updated the first sentence as above + rebased.